src/BTConnectionManager.h

```diff
@@ -80,6 +85,9 @@ public Q_SLOTS:
 private:
     class Private;
     Private* d;
+    AppSettings *m_appSettings;
```

Comment:
This goes entirely against how PImpl should work. Both this variable, and the function underneath, need to be moved into the Private class.

Reply:
Yes, my apologizes. I will change it.